lib/src/render/image/image_node_widget.dart

```diff
@@ -138,30 +141,50 @@ class ImageNodeWidgetState extends State<ImageNodeWidget> with SelectableMixin {
   }
 
   Widget _buildResizableImage(BuildContext context) {
-    final networkImage = Image.network(
-      widget.src,
-      width: _imageWidth == null ? null : _imageWidth! - _distance,
-      gaplessPlayback: true,
-      loadingBuilder: (context, child, loadingProgress) {
-        if (loadingProgress == null ||
-            loadingProgress.cumulativeBytesLoaded ==
-                loadingProgress.expectedTotalBytes) {
-          return child;
+    Image? image;
+
```

Comment:
```suggestion
    Image? image;
    final width = _imageWidth == null ? null : _imageWidth! - _distance; 

```
You can move this ternary operation outside of the switch so it's the same for both cases, you don't have to though.